### PR TITLE
Bump dependency versions

### PR DESCRIPTION
**Pull request by xylo04**
No strong motivation, just housekeeping.

### CODE REVIEW

**Comment by xylo04** [comment]
@martinhpedersen is "Travis CI - Branch" supposed to kick off automatically?

File: go.mod [hunk 1]
```diff
@@ -1,8 +1,8 @@
 module github.com/la5nta/wl2k-go
 
-go 1.11
+go 1.14
```
Comment (martinhpedersen) [inline go.mod:3]: Please revert this  - ref https://github.com/la5nta/pat/pull/222#pullrequestreview-525832837.

**Review by martinhpedersen — CHANGES_REQUESTED** [review]
@xylo04 Thanks!

I actually don't know why the "Travis CI - Branch" is listed as a required status check here. May be related to some housekeeping I've done regarding CI a few months ago. I will change the base branch to `develop` and see if that helps.

PS: Do you mind reverting the Go 1.11 -> 1.14 change? Ref review comment.

File: go.mod (hunk 1 above)
Comment (xylo04) [reply go.mod]: Done! I should've remembered to do that here, too. Thanks

**Review by martinhpedersen — APPROVED** [review]
Perfect! Thank you 😊

**Pull request merged**